helm/grafana/Chart.yaml

```diff
@@ -1,28 +1,27 @@
 apiVersion: v2
-appVersion: 7.3.5
+name: grafana
+version: 6.24.1
+appVersion: 8.4.2
```

Comment:
This is a major bump? What are the breaking changes?

Reply:
https://grafana.com/docs/grafana/latest/release-notes/release-notes-8-0-0/#breaking-changes Nothing really important